sled-agent/config-reconciler/src/internal_disks.rs

```diff
@@ -481,6 +491,14 @@ impl From<&'_ Disk> for InternalDiskDetails {
     fn from(disk: &'_ Disk) -> Self {
         // Synthetic disks panic if asked for their `slot()`, so filter
         // them out first.
```

Comment:
I think this comment needs to move below the new code.

Reply:
done, thanks